extras/AFC_led.py

```diff
@@ -128,21 +128,31 @@ def lookahead_bgfunc(print_time):
             else:
                 set_color_fn = self.led_helper.set_color
                 check_transmit_fn = self.led_helper.check_transmit
-            set_color_fn(index, colors)
+
+            if "-" in index:
+                start, end = map(int, index.split("-"))
+                for i in range(start, end + 1):
+                    set_color_fn(i, colors)
+            else:
+                set_color_fn(int(index), colors)
+
             if transmit:
                 check_transmit_fn(print_time)
         toolhead = self.printer.lookup_object('toolhead')
         toolhead.register_lookahead_callback(lookahead_bgfunc)
 
     def turn_off_leds(self):
-        for i in range(self.led_helper.led_count):
-            self.led_change( i, "0,0,0,0", False)
+        for lane in self.AFC.lanes:
```

Comment:
Recently these variables were updated to be lowercase, please update. Also can you please test to make sure `TURN_OFF_AFC_LED` and `TURN_ON_AFC_LED` macros work correctly
```suggestion
        for lane in self.afc.lanes:
```